File: yurt-system/yurt-system-server/src/main/java/com/github/liulus/yurt/system/model/dto/MenuQo.java
package com.github.liulus.yurt.system.model.dto;

import com.github.liulus.yurt.convention.data.PageQuery;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * User : liulu
 * Date : 2018/1/15 10:41
 * version $Id: MenuQo.java, v 0.1 Exp $
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class MenuQo extends PageQuery {

    private static final long serialVersionUID = 1204535134806916841L;

    private Long id;

    private Long parentId = 0L;

    private String menuCode;

    private String keyword;


}

File: yurt-system/yurt-system-server/src/main/kotlin/com/github/liulus/yurt/system/model/dto/MenuQo.kt
package com.github.liulus.yurt.system.model.dto

import com.github.liulus.yurt.convention.data.PageQuery

/**
 * User : liulu
 * Date : 2018/1/15 10:41
 * version $Id: MenuQo.java, v 0.1 Exp $
 */
class MenuQo : PageQuery() {
    var id: Long? = null
    var parentId = 0L
    var menuCode: String? = null
    var keyword: String? = null
    
}